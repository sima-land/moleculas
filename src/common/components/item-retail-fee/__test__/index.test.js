import React from 'react';
import { mount } from 'enzyme';
import { ItemRetailFee } from '../index';

describe('<ItemRetailFee />', () => {
  it('should render without props', () => {
    const wrapper = mount(<ItemRetailFee />);
    expect(wrapper).toMatchSnapshot();
  });
  it('should render with props', () => {
    const wrapper = mount(<ItemRetailFee isFixed isSmaller price={15} minQty={20} unit='R' />);
    expect(wrapper).toMatchSnapshot();
  });
});
