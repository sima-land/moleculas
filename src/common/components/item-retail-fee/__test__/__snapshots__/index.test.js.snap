// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ItemRetailFee /> should render with props 1`] = `
<ItemRetailFee
  isFixed={true}
  isSmaller={true}
  minQty={20}
  price={15}
  unit="R"
>
  <div
    className="wrapper fixed-width small"
  >
    <Box
      marginTop={4}
    >
      <div
        className="M-t__4"
      >
        <Memo(Text)
          color="gray38"
          lineHeight={16}
          size={12}
        >
          <span
            className="size-12 line-height-16 color__gray38"
          >
            Комплектация
            <span
              className="no-wrap"
            >
               + 
              <Price
                className="price"
                value={15}
              >
                <span
                  className="price price"
                >
                  15
                </span>
              </Price>
            </span>
            <span>
              при покупке до 
              20
               
              R
            </span>
          </span>
        </Memo(Text)>
      </div>
    </Box>
  </div>
</ItemRetailFee>
`;

exports[`<ItemRetailFee /> should render without props 1`] = `
<ItemRetailFee>
  <div
    className="wrapper"
  >
    <Box
      marginTop={4}
    >
      <div
        className="M-t__4"
      >
        <Memo(Text)
          color="gray38"
          lineHeight={20}
          size={14}
        >
          <span
            className="size-14 line-height-20 color__gray38"
          >
            Комплектация
            <span
              className="no-wrap"
            >
               + 
              <Price
                className="price"
              >
                <span
                  className="price price"
                />
              </Price>
            </span>
          </span>
        </Memo(Text)>
      </div>
    </Box>
  </div>
</ItemRetailFee>
`;
